core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -7,32 +7,43 @@
  */
 package io.camunda.migrator;
 
-import io.camunda.client.CamundaClient;
-import io.camunda.client.api.command.ModifyProcessInstanceCommandStep1;
-import io.camunda.client.api.command.ModifyProcessInstanceCommandStep1.ModifyProcessInstanceCommandStep3;
-import io.camunda.client.api.response.ActivatedJob;
-import io.camunda.migrator.history.IdKeyDbModel;
-import io.camunda.migrator.history.IdKeyMapper;
+import static io.camunda.migrator.HistoryMigrator.BATCH_SIZE;
```

Comment:
please ignore the import re-order, I will get rid of that in the second iteration.